site/shared/src/main/scala/shared/Contribution.scala

```diff
@@ -0,0 +1,22 @@
+package shared
+
+case class Contributions(
+  commits:      List[Contribution],
+  contributors: List[Contributor]
+)
+
+case class Contribution(
+  sha:        String,
+  message:    String,
+  date:       String,
+  url:        String,
+  login:      String,
+  avatar_url: String,
```

Comment:
Same as the comment earlier-- Are we using snake_case here because these values map to HTTP API fields? I would prefer that we consistently use camelCase, if possible.